pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -880,13 +919,35 @@ func VerifyUsableExportTarget(
 	}
 
 	if !errors.Is(err, cloudimpl.ErrFileDoesNotExist) {
-		return errors.Wrapf(err, "%s returned an unexpected error when checking for the existence of %s file", readable, BackupManifestCheckpointName)
+		return errors.Wrapf(err,
+			"%s returned an unexpected error when checking for the existence of %s file",
+			readable, BackupManifestCheckpointName)
 	}
 
-	if err := writeBackupManifest(
-		ctx, settings, exportStore, BackupManifestCheckpointName, encryption, &BackupManifest{},
-	); err != nil {
-		return errors.Wrapf(err, "cannot write to %s", readable)
+	return nil
+}
+
+// verifyWritableDestination writes a test file, verifying that the location is
+// writable. This method will do a best-effort clean up of the temporary file.
+// We don't require DELETE permissions on their backup directory, so we do not
+// enforce that this file be deleted.
+func verifyWriteableDestination(
+	ctx context.Context, exportStore cloud.ExternalStorage, writeable string,
+) error {
+	// Write arbitrary bytes to a sentinel file in the backup directory to ensure
+	// that we're able to write to this directory.
+	if err := exportStore.WriteFile(ctx, BackupSentinelWriteFile, bytes.NewReader([]byte{0xFF})); err != nil {
```

Comment:
personally I regret any chance I don't take to use an emoji for a magic/header/sentinel. It is obviously dealer's choice here, but I will say that ✇ exists.

Reply:
Haha - I'll byte.